pipeline_dp/budget_accounting.py

```diff
@@ -171,6 +235,9 @@ def compute_budgets(self):
             logging.warning("No budgets were requested.")
             return
 
+        if len(self._scopes) > 0:
```

Comment:
Nit: Python style for checking non-emptyness:
  `if self._scopes`

(on emptyness   `if not self._scopes` )